apps/docs/src/basics/typography.md

```diff
@@ -24,6 +24,8 @@ _custom component for displaying font definitions_
 
 ### Headlines
 
+For the technical implementation of the headlines, please see the [technical documentation](/development/OnyxHeadline).
```

Comment:
[question] is it technically possible to provide  a link back inside the storybook file for OnyxHeadline? so we link in a loop -> when you jump from here, you can go back wihtout using browser-back ^^ and when you only look at the dev side you can also go over to the UX perspective :) not a must-have though :)

probably problematic for devs that only start the storybook though because it runs on a different port/URL ^^

anyways, if we wanna do that, it can clearly be a separate PR :)